src/sgx/libpoet_bridge/poet.cpp

```diff
@@ -291,6 +255,18 @@ poet_err_t Poet_GetEnclaveCharacteristics(
            inEnclaveBasenameLength,
            hexString.c_str(),
            hexString.length());
+
+        std::reverse(epidGroup, epidGroup + sizeof(epidGroup));
```

Comment:
[Question]: epidGroup is of type sgx_epid_group_id_t  and sgx_epid_group_id_t  is array of uint8_t with size 4. sizeof(epidGroup) would then return 1 byte * 4 length which is 4. So intent here is to reverse order elements in array epidGroup. Could you please correct me if I didn't understand something here?

Reply:
Yes, the understanding is correct

Reply:
Could you please add a comment here along with unit test correction patch? (something like changing the epidGroup to big endian)